src/snowflake/snowpark/modin/plugin/__init__.py

```diff
@@ -103,4 +102,4 @@ def isocalendar(self):  # type: ignore
     return DataFrame(query_compiler=self._query_compiler.dt_isocalendar())
 
 
-DatetimeProperties.isocalendar = isocalendar
+modin.pandas.series_utils.DatetimeProperties.isocalendar = isocalendar
```

Comment:
@sfc-gh-joshi let's first merge this to clear the daily job, and i assume you will have another pr to do it in a more clear way